src/app/Http/Controllers/BusinessTripController.php

```diff
@@ -11,21 +25,23 @@ class BusinessTripController extends Controller
      * Returning view with details from all trips
      */
     public function index() {
-        return view();
+        $trip = BusinessTrip::all();
+        return view('business-trips.show', ['trip' => $trip]);
```

Comment:
Neviem, ci ten view bude takto fungovat a urcite nechceme vzdy zobrazovat vsetky cesty v DB, len tie relevantne pre daneho usera.

Reply:
no, nazvala by som tu premennu urcite $trips, lebo je ich vela. A tam by to chcelo check na to, kto je prihlaseny. ak admin, poslu sa vsetky, a ak nie admin, poslu sa iba cesty konkretneho pouzivatela. A tiez ak je prihlaseny admin, chcelo by to poslat aj zoznam pouzivatelov

Reply:
vlastne to som si ani nevsimla, ale ten view tu nie je spravny, ma tu byt 'dashboard'